tests/mail/tests.py

```diff
@@ -730,14 +730,14 @@ def test_sanitize_address(self):
             (
                 ('A name', 'to@example.com'),
                 'utf-8',
-                '=?utf-8?q?A_name?= <to@example.com>',
+                'A name <to@example.com>',
```

Comment:
There should never be a reason to encode ascii to something else. While technically valid it makes mails bigger (especially if at one point we decide to call sanitize address for all addresses)

Reply:
I think the last time this code changed, we did this because it was the way it was done before, but I also support not encoding already correct ascii. Chances are no one will complain that this changed.

Reply:
Maybe we should add a short note about this change in release notes :thinking: